iroh-metrics/src/lib.rs

```diff
@@ -7,8 +7,9 @@ pub mod metrics;
 pub mod core;
 
 /// Expose iroh metrics
+/// todo: remove pub export
```

Comment:
what's the alternative here? is this because we are considering it temporary™?

Reply:
Export just the one fn we need, not the entire module?

Reply:
Yeah I dropped this in favor of keeping everything in line as it was.